src/pages/map.tsx

```diff
@@ -140,13 +140,15 @@ const MapPage = ({ schedule }) => {
 
   const setInitialMapBound = (mapBox: MapboxGl.Map) => {
     // eslint-disable-next-line @typescript-eslint/no-floating-promises
-    window.navigator.permissions.query({ name: 'geolocation' }).then(data => {
-      const permission = data.state === 'granted';
-      setGetGeoPermission(permission);
-      if (!permission) {
-        mapBox.fitBounds(getMapBounds(coordinates), { padding: 100 });
-      }
-    });
+    if (window && window.navigator && window.navigator.permissions) {
```

Comment:
maybe just `window.navigator.permission` or `'permissions' in navigator`

```suggestion
    if ('permissions' in navigator) {
```